src/test_driver/nrfconnect/prj.conf

```diff
@@ -15,7 +15,7 @@
 #
 
 # Application stack size
-CONFIG_MAIN_STACK_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=8704
```

Comment:
Where is the bigger stack use?

Reply:
I think my real question here is: why are we using more stack now?  stack-usage is on a per-function basis, right?  The only thing on the stack in `TestEventOverflow` is a `TestContext`.  I guess that has a FabricTable member and that got bigger by the one extra FabricInfo?  If so, that answers my question here, I think.